### PR TITLE
Support reactive scan/query operations.

### PR DESCRIPTION
**Pull request by roimenashe**

### CODE REVIEW

File: src/main/java/com/aerospike/mapper/tools/ReactiveAeroMapper.java [hunk 1]
```diff
@@ -489,6 +492,159 @@ private ReactiveAeroMapper(@NotNull IAerospikeReactorClient reactorClient) {
...
+        String setName = entry.getSetName();
+
+        AtomicBoolean userTerminated = new AtomicBoolean(false);
+
+        return reactorClient.scanAll(policy, namespace, setName)
+            .map(keyRecord -> {
+                T object = this.getMappingConverter().convertToObject(clazz, keyRecord.record);
+                if (!userTerminated.get()) {
+                    try {
+                        if (!processor.process(object)) {
```
Comment (reugn) [inline src/main/java/com/aerospike/mapper/tools/ReactiveAeroMapper.java:581]: You don't really need the processor here. Just convert to an Object and return. Also, try to use `Flux.handle` to return an error downstream and skip further elements.

File: src/main/java/com/aerospike/mapper/tools/ReactiveAeroMapper.java [hunk 2]
```diff
@@ -489,6 +492,159 @@ private ReactiveAeroMapper(@NotNull IAerospikeReactorClient reactorClient) {
...
+        statement.setNamespace(entry.getNamespace());
+        statement.setSetName(entry.getSetName());
+
+        AtomicBoolean userTerminated = new AtomicBoolean(false);
+
+        return reactorClient.query(policy, statement)
+                .map(keyRecord -> {
+                    T object = this.getMappingConverter().convertToObject(clazz, keyRecord.record);
+                    if (!userTerminated.get()) {
+                        if (!processor.process(object)) {
```
Comment (reugn) [inline src/main/java/com/aerospike/mapper/tools/ReactiveAeroMapper.java:640]: The same as in the `scan` method.

File: src/main/java/com/aerospike/mapper/tools/ReactiveAeroMapper.java (hunk 1 above)
Comment (roimenashe) [reply src/main/java/com/aerospike/mapper/tools/ReactiveAeroMapper.java]: Modified, please review again.

File: src/main/java/com/aerospike/mapper/tools/ReactiveAeroMapper.java (hunk 2 above)
Comment (roimenashe) [reply src/main/java/com/aerospike/mapper/tools/ReactiveAeroMapper.java]: Modified, please review again.

**Review by reugn — APPROVED** [review]

**Pull request merged**